tox.ini

```diff
@@ -116,13 +125,14 @@ commands =
          ext/opentelemetry-ext-http-requests/tests/ \
          ext/opentelemetry-ext-jaeger/src/opentelemetry \
          ext/opentelemetry-ext-jaeger/tests/ \
-         ext/opentelemetry-ext-opentracing-shim/src/ \
-         ext/opentelemetry-ext-opentracing-shim/tests/ \
          ext/opentelemetry-ext-pymongo/src/opentelemetry  \
          ext/opentelemetry-ext-pymongo/tests/ \
          ext/opentelemetry-ext-wsgi/tests/ \
          examples/opentelemetry-example-app/src/opentelemetry_example_app/ \
          examples/opentelemetry-example-app/tests/
+  ; it has to be in a different instance because otherwise gives import errors
```

Comment:
Have you tried putting just `tests` in a separate invocation?

Reply:
Just tried, same problem.

Reply:
what are the import errors?

Reply:
When I use "python scripts/eachdist.py lint --check-only" these are the errors I see.
I don't remember if I went into the details or not, will check more later on.

Posting the error messages here to see if somebody has any idea at this point.

```
************* Module tests.testbed.test_late_span_finish.test_threads
ext/opentelemetry-ext-opentracing-shim/tests/testbed/test_late_span_finish/test_threads.py:6:0: E0401: Unable to import 'tests.testbed.otel_ot_shim_tracer' (import-error)
************* Module tests.testbed.test_late_span_finish.test_tornado
ext/opentelemetry-ext-opentracing-shim/tests/testbed/test_late_span_finish/test_tornado.py:5:0: E0401: Unable to import 'tests.testbed.otel_ot_shim_tracer' (import-error)
ext/opentelemetry-ext-opentracing-shim/tests/testbed/test_late_span_finish/test_tornado.py:6:0: E0401: Unable to import 'tests.testbed.testcase' (import-error)

```